app/views/invitation_mailer/invite_message.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1><%= "#{User.find(@user.invited_by_id).full_name} has invited you to join the team as #{@role.name}" %></h2>
+<h1><%= "#{@invitation_sender.full_name} has invited you to join the team as #{@role.name}" %></h2>
```

Comment:
Templates should not pull in data from the data base. This makes it easier to test. The inviting user does not need to be persisted.